tsconfig.json

```diff
@@ -45,6 +45,7 @@
   "include": [
       "exports/**/*.ts",
       "typedoc-index.ts",
-      "pal/**/*.ts"
+      "pal/**/*.ts",
+      "editor/exports/**/*.ts"
```

Comment:
please don't introduce this included item

Reply:
when includes this editor exports item, vscode auto import may import from editor exports